tools/install.sh

```diff
@@ -155,7 +156,16 @@ function sync_dependencies {
 		 [ "$LINUX_VERSION" == "Debian"  ] || \
 		 [ "$LINUX_VERSION" == "Kali"    ] || \
 		 [ "$LINUX_VERSION" == "Ubuntu"  ]; then
-		apt-get install -y libfreetype6-dev python3-dev python3-pip pkg-config
+		apt-get install -y libfreetype6-dev python3-dev pkg-config
+		if ! python3 -m pip --version; then
+			if apt-get install python3-pip; then
+				echo "Installed python3-pip via apt-get"
+			else
+				curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
```

Comment:
Is it possible to validate the data coming from PyPa.io so we're not just executing some random script as root? Also can we either delete the file after we're done or pipe it straight into Python?